scripts/prepare-environment-k3d.sh

```diff
@@ -89,4 +87,6 @@ if [ ${retry} -ge ${maxRetries} ]; then
   exit 1
 fi
 
+"${EPINIO_BINARY}" info
```

Comment:
Strange but I got this:

```
✔️  Login successful

❌  error retrieving Epinio environment information: cannot parse JSON response: 'Bad Gateway': invalid character 'B' looking for beginning of value
make: *** [Makefile:189: prepare_environment_k3d] Error 255
```

How is it possible to successfully login if the server is not ready yet. Tryiing `epinio info` manually after the error, works.

Reply:
Maybe we can define a function like this:

```
function retry {
  retry=0
  maxRetries=$1
  retryInterval=$2
  local result
  until [ ${retry} -ge ${maxRetries} ]
  do
    result=$(eval "$3") && break
    retry=$[${retry}+1]
    sleep ${retryInterval}
  done

  if [ ${retry} -ge ${maxRetries} ]; then
    echo "Failed to run "$3" after ${maxRetries} attempts!"
    exit 1
  fi

  echo $result
}
```

which can then be called like:

```
retry 5 1 "${epinio} info"
```

(copied it from another script I had locally)

Reply:
Done! :+1: